### ISSUE TITLE
Extra "back-end" information of the site

### ISSUE THREAD

**Issue by truls1502** [pantheon-systems/localdev-issues#8]
When I have pulled the site. I can see below:
- Local site
- Cloud site
- Code
- Database

It would be great to add extra information such as:
- Site information
   - Created when
   - Service level
   - Upstream name (Later it would be awesome to change the upstream from the localdev-app)
   - Branch name
   - Framework
   - Region
- Back-end information
   - PHP version
   - Composer version
   - Drush version
   - Status such as in `#dev/status`
- Git
   - Last 10 log (Preview only)
- Link to errorlog

The idea is to give the developer easier to track and find the issue when contact Pantheon support or someone who doesn't like the terminal.

**Comment by joshkoenig** [comment]
Makes sense to pull some of this into 1.0 scope, but probably not for the upcoming beta.
